app/schedulers/complete_mentorship_cron_job.py

```diff
@@ -12,15 +12,16 @@ def complete_overdue_mentorship_relations_job():
     with application.app_context():
         from app.utils.enum_utils import MentorshipRelationState
         from app.database.models.mentorship_relation import MentorshipRelationModel
-        all_relations = MentorshipRelationModel.query.all()
+        all_relations = filter(
+            lambda relation: relation.state == MentorshipRelationState.ACCEPTED
+                             and relation.end_date < current_date_timestamp,
+            MentorshipRelationModel.query.all())
 
         current_date_timestamp = datetime.now().timestamp()
 
         for relation in all_relations:
+            relation.state = MentorshipRelationState.COMPLETED
+            relation.save_to_db()
 
-            if relation.state == MentorshipRelationState.ACCEPTED and relation.end_date < current_date_timestamp:
-                relation.state = MentorshipRelationState.COMPLETED
-                relation.save_to_db()
-
-            # for tests purposes
-            # print('{} | hola hola hola hola hola'.format(datetime.now()))
+        # for tests purposes
```

Comment:
this need additional spacing/tab

Reply:
![Screenshot (93)](https://user-images.githubusercontent.com/34242059/72184742-6dc54080-33f1-11ea-90ce-3cc8fefc1b9a.png)

Hmm... Correct, if I'm wrong, but it is not in the if statement anymore. Why does it need an extra tab?

Reply:
it was in the loop not in if

Reply:
@annabauza oh yes! You are totally right! Sorry, my bad :)